sdk/identity/identity/test/internal/node/azureCliCredential.spec.ts

```diff
@@ -1,40 +1,34 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
-
-import type Sinon from "sinon";
-import { createSandbox } from "sinon";
-
-import { AzureCliCredential } from "../../../src/credentials/azureCliCredential";
+import { AzureCliCredential } from "../../../src/credentials/azureCliCredential.js";
 import type { GetTokenOptions } from "@azure/core-auth";
-import { assert } from "@azure-tools/test-utils";
 import child_process from "child_process";
+import { describe, it, assert, expect, vi, beforeEach, afterEach } from "vitest";
 
 describe("AzureCliCredential (internal)", function () {
-  let sandbox: Sinon.SinonSandbox | undefined;
   let stdout: string = "";
   let stderr: string = "";
   let azArgs: string[][] = [];
   let azOptions: { cwd: string; shell: boolean }[] = [];
 
   beforeEach(async function () {
-    sandbox = createSandbox();
     azArgs = [];
     azOptions = [];
-    sandbox
-      .stub(child_process, "execFile")
-      .callsFake((_file, args, options, callback): child_process.ChildProcess => {
+    vi.spyOn(child_process, "execFile").mockImplementation(
+      (_file, args, options, callback): child_process.ChildProcess => {
```

Comment:
Could probably just import the type ChildProcess perhaps?

Reply:
fixed